MODIS_API/k_means_method.py

```diff
@@ -10,27 +10,39 @@ def create_labels_colors(input_image, clusters=5):
     @return: colors_array, classification array of length clusters OR empty array on failure
     @return: labels_array, array of category of input array OR empty array on failure
     """
+    
     if (not isinstance(clusters, int) or not isinstance(input_image, list)):
         return ([], [])
 
+    reshaped_image = reshape_data(input_image, ratio_array1, ratio_array2)
+    
+
     kmeans = KMeans(clusters)
-    kmeans.fit(input_image)
+    kmeans.fit(reshaped_image)
 
     colors_array = kmeans.cluster_centers_.astype(int)
 
     labels_array = kmeans.labels_
 
-    return (colors_array, labels_array)
+    return (reshaped_image, colors_array, labels_array)
```

Comment:
New return outputs the reshaped_image array as 2D flattened image with (R,G,B,ratio1, ratio2) data